crds/cloudbuild_v1beta1_cloudbuildworkerpool.yaml

```diff
@@ -23,6 +23,8 @@ metadata:
     cnrm.cloud.google.com/stability-level: beta
   name: cloudbuildworkerpools.cloudbuild.cnrm.cloud.google.com
 spec:
+  conversion:
+    strategy: None
```

Comment:
Ideally we shall use a webhook here because the cbwp alpha to beta changes a field
`spec.privatePoolV1Config.networkConfig.peeredNetworkIpRange` --> `spec.privatePoolV1Config.networkConfig.peeredNetworkIPRange`